local-cli/server/middleware/getDevToolsMiddleware.js

```diff
@@ -24,7 +25,42 @@ function getChromeAppName() {
   }
 }
 
-module.exports = function(options, isDebuggerConnected) {
+function launchChromeDevTools(options) {
+    var debuggerURL = 'http://localhost:' + options.port + '/debugger-ui';
+    console.log('Launching Dev Tools...');
+    opn(debuggerURL, {app: [getChromeAppName()]}, function(err) {
+        if (err) {
+            console.error('Google Chrome exited with error:', err);
+        }
+    });
+}
+
+function escapePath(path) {
+  return path.indexOf(' ') > -1
```

Comment:
Do other whitespace characters (e.g. `\t`) need to be quoted as well? What if we simply quote all paths, regardless of whether they contain whitespace?